mango_library/negotiation/termination.py

```diff
@@ -121,12 +121,12 @@ def setup(self):
         )
 
     def on_send(
```

Comment:
signature for on_send subscriptions is not content, receiver_addr: AgentAddress, **kwargs